### DISCUSSION TITLE
Got error Cannot find module node_modules\lodash\debounce

### DISCUSSION THREAD

**Discussion by whytrno** [ueberdosis/tiptap#3712] [General Questions]
Hallo, can someone help me to solve this issue?
![image](https://user-images.githubusercontent.com/70625779/217119792-8ebb2981-0bfd-43e5-b23a-3d007cd40cc5.png)
```
Error: Cannot find module 'D:\Wahyu\Project\Next Js\nowted\node_modules\lodash\debounce' imported from D:\Wahyu\Project\Next Js\nowted\node_modules\@tiptap\extension-bubble-menu\dist\index.js
Did you mean to import lodash/debounce.js?
```
For now i solve this issue with, change node_modules\@tiptap\extension-bubble-menu\dist\index.js file like this
From:
```
import debounce from "lodash/debounce"
```
To:
```
import debounce from "lodash/debounce.js";
```

But its annoying when i change my device, or maybe when i deploy my app. I should manually change that file again.

**Comment by max-nextcloud** [comment]
It's been reported in #3708. There are some workarounds proposed. The fix has been developed - just needs to be shipped.
